src/frontend/src/icp-eth/components/fee/FeeAmountDisplay.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+	import { nonNullish } from '@dfinity/utils';
+	import { fade, slide } from 'svelte/transition';
+	import { formatToken } from '$lib/utils/format.utils';
+	import { EIGHT_DECIMALS } from '$lib/constants/app.constants';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+	import { balancesStore } from '$lib/stores/balances.store.js';
+	import { BigNumber } from '@ethersproject/bignumber';
+	import type { TokenId } from '$lib/types/token';
+	import { onDestroy } from 'svelte';
+
+	export let fee: BigNumber;
+	export let feeSymbol: string;
+	export let feeTokenId: TokenId;
+
+	const balance = nonNullish($balancesStore)
+		? $balancesStore[feeTokenId]?.data ?? BigNumber.from(0n)
+		: undefined;
+
+	let insufficientFeeFunds = false;
+	let timer: NodeJS.Timeout | undefined;
+
+	$: {
+		timer = setTimeout(() => {
+			insufficientFeeFunds = nonNullish(balance) && balance.lt(fee);
+		}, 250);
+	}
+
+	onDestroy(() => clearTimeout(timer));
+</script>
+
+<div transition:fade>
```

Comment:
There is no if so fade as no effect here

Reply:
I'm really suprised. I was under the impression that Svelte requires any animation tag to be contained within a direct parent if. Also thought there was a lint check for that. Probably misunderstood it it works. Thanks for the feebdack.